routes/api/v1/favorites.js

```diff
@@ -54,5 +58,54 @@ router.post("/", function(req, res) {
     })
 
     .catch(error => res.send({error}));
-})
+});
+
+router.get("/", function(req, res) {
+  User.findOne({
+    where: {
+      api_key: req.body.api_key
+    },
+    include: [Location]
+  })
+    .then( user => {
+      if(!user){
+        res.sendStatus(401)
+      } else{
+        let favorites = [];
+        user.getLocations()
+          .then(locations => {
+            var userLocations = locations;
+            for (i = 0; i < locations.length; i++) {
```

Comment:
Instead of a counter, you could try using map to iterate through all your locations and produce a new array. Might make this slightly more readable!

Reply:
Great idea, thanks! I did locations.map which I think it's easier to understand